scripts/unit-tests/windows/ovs-build.ps1

```diff
@@ -0,0 +1,408 @@
+function Start-CompressLogs ( $logsPath ) {
+    $logfiles = Get-ChildItem -File -Recurse -Path $logsPath | Where-Object { $_.Extension -ne ".gz" }
+    foreach ($file in $logfiles) {
+        $filename = $file.name
+        $directory = $file.DirectoryName
+        $extension = $file.extension
+        if (!$extension) {
+            $name = $file.name + ".txt"
+        }
+        else {
+            $name = $file.name
+        }
+        &7z.exe a -tgzip "$directory\$name.gz" "$directory\$filename" -sdel
+    }
+}
+
+function Set-RemoteVars {
+    Set-Variable -Name server -Value "10.20.1.14" -Scope Script
+    Set-Variable -Name user -Value "logs" -Scope Script
+    Set-Variable -Name key -Value "C:\scripts\ssh\norman.ppk" -Scope Script
+    Set-Variable -Name remotelogDir -Value "/srv/logs/ovs" -Scope Script
+    Set-Variable -Name remotemsiDir -Value "/srv/dl/ovs" -Scope Script
+}
+
+function Start-SSHCMD ($server, $user, $key, $cmd) {
+    write-host "Running ssh command $cmd on remote server $server"
+#        &plink -batch $server -l $user -i $key $cmd
+    echo Y | plink.exe $server -l $user -i $key $cmd
+}
+
+function Start-SCP ($server, $user, $key, $localPath, $remotePath) {
+    write-host "Starting copying $localPath to remote location ${server}:${remotePath}"
+#        &pscp -batch -scp -r -i $key $localPath $user@${server}:${remotePath}
+    echo Y | pscp.exe -scp -r -i $key $localPath $user@${server}:${remotePath}
+}
+
+function Start-CreateRemoteDir {
+    Set-RemoteVars
+    $logCMD = "mkdir -p $remotelogDir/$commitID"
+    $msiCMD = "mkdir -p $remotemsiDir/$commitID"
+    Start-SSHCMD $server $user $key $logCMD
+    Start-SSHCMD $server $user $key $msiCMD
+}
+
+function Start-RemoteLogCopy ($locallogPath) {
+    Set-RemoteVars
+    $remotelogPath = "$remotelogDir/$commitID"
+    write-host "Started copying logs for commit ID $commitID unit tests to remote location ${server}:${remotelogPath}"
+    Start-SCP $server $user $key $locallogPath $remotelogPath
+}
+
+function Start-RemoteMsiCopy ($localmsiPath) {
+    Set-RemoteVars
+    $remotemsiPath = "$remotemsiDir/$commitID"
+    write-host "Started copying generated msi for commit ID $commitID to remote location ${server}:${remotemsiPath}"
+    Start-SCP $server $user $key $localmsiPath $remotemsiPath
+}
+
+function Start-Cleanup {
+    $ovsprocess = Get-Process ovs* -ErrorAction SilentlyContinue
+    $ovnprocess = Get-Process ovn* -ErrorAction SilentlyContinue
+    if ($ovsprocess) {
+        Stop-Process -name ovs*
+    }
+    if ($ovnprocess) {
+        Stop-Process -name ovn*
+    }
+    cd $buildDir
+    CheckRemoveDir $commitDir
+}
+
+function Start-MailJob ($user, $pass, $status) {
+    $uri = "http://10.20.1.3:8080/job/ovs-email-job/buildWithParameters?token=b204eee759ab38ebb986d223f6c5b4ce&commitid=$commitID&status=$status"
+    $pair = "${user}:${pass}"
+    $bytes = [System.Text.Encoding]::ASCII.GetBytes($pair)
+    $base64 = [System.Convert]::ToBase64String($bytes)
```

Comment:
Removed this function and instead, now the email job gets its parameters from a file and is started after the unit-tests job finished not matter the outcome